lib/ja_serializer/serializer.ex

```diff
@@ -1,47 +1,47 @@
 defmodule JaSerializer.Serializer do
   @moduledoc """
-  Define a serialization schema.
+  A Behaviour for defining JSON-API.org spec complaint payloads.
```

Comment:
super nitpick, but you're not consistent in the docs with how you refer to the JSON API spec. In some places it's "JSONAPI.org", others it's "JSONAPI spec".